mapping_config: "$(find fastdem)/config/default.yaml"

map:
  width: 15.0       # [m]
  height: 15.0      # [m]
  resolution: 0.1   # [m/cell]

topics:
  input_scan: "/velodyne/points"
  # input_scan: "/camera_front/pointcloud/points"
  publish_rate: 10.0  # [Hz]

tf:
  base_frame: "base_link"
  map_frame: "map"
  max_wait_time: 0.1  # [sec] Wait time for transform to arrive
  max_stale_time: 0.1 # [sec] Max time difference between query and latest transform

logger:
  level: "info" # info, warn, error, debug
